src/Commands/BackupCommand.php

```diff
@@ -59,6 +69,15 @@ public function handle()
 
             consoleOutput()->comment('Backup completed!');
         } catch (Exception $exception) {
+            if ($this->tries > 1 && $this->currentTry < $this->tries) {
```

Comment:
Conditionals like this are not that readable. Could you refactor this to a dedicated method with a good name, that returns a boolean and uses early returns.